frontend/app/components/shelterDetail.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Col, Media } from 'react-bootstrap'
+import email from '../../static/email.png'
+import phone from '../../static/phone.png'
+import location from '../../static/location.png'
+
+class ShelterDetail extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  fillEmptyValues() {
+    const {cat} = this.props
+    _.forEach(cat, function(value, key) {
+      if(value.length == undefined && key != 'photos'){
```

Comment:
you want to use `===` and `!==` here. it's an annoying JS quirk but you almost never want to use `==` and `!=` in JS